modules/Calendar/Cal.js

```diff
@@ -730,63 +730,67 @@ CAL.cut_record = function (id) {
 CAL.init_edit_dialog = function (params) {
     CAL.editDialog = false;
     var rd = CAL.get("cal-edit");
-    var content = CAL.get("edit-dialog-content");
-    if (CAL.dashlet && rd) {
-        document.getElementById("content").appendChild(rd);
-    }
-    rd.style.width = params.width + "px";
-    content.style.height = params.height + "px";
-    content.style.overflow = "auto";
-    content.style.padding = "0";
-    CAL.editDialog = new YAHOO.widget.Dialog("cal-edit", {
-        draggable: true,
-        visible: false,
-        modal: true,
-        close: true,
-        y: 1,
-        zIndex: 10
-    });
-    var listeners = new YAHOO.util.KeyListener(document, {keys: 27}, {
-        fn: function () {
-            CAL.editDialog.cancel();
-        }
-    });
-    CAL.editDialog.cfg.queueProperty("keylisteners", listeners);
-    CAL.editDialog.cancelEvent.subscribe(function (e, a, o) {
-        CAL.close_edit_dialog();
-    });
-    rd.style.display = "block";
-    CAL.editDialog.render();
-    rd.style.overflow = "auto";
-    rd.style.overflowX = "hidden";
-    rd.style.outline = "0 none";
-    rd.style.height = "auto";
+    //var content = CAL.get("edit-dialog-content");
```

Comment:
????? Why so much commented out code.  When making a PR please check what you are commenting.